### PR TITLE
Update reputation.md

### PR DESCRIPTION
**Pull request by jessiwright**

### CODE REVIEW

File: docs/hackers/reputation.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Not Applicable | -5
 Duplicate of a resolved report submitted after the report is made public | -5
 Duplicate of a N/A report | -5
 Spam | -10
+Bug Reopened | -7
```
Comment (martijnrusschen) [inline docs/hackers/reputation.md:30]: Can you verify this? I believe this activity reverts the previously given reputation.

Comment (jessiwright) [reply docs/hackers/reputation.md]: Corrected

**Comment by jessiwright** [comment]
@martijnrusschen Can you re-review please?

**Review by martijnrusschen — APPROVED** [review]
This technically applies to other states as well

**Pull request merged**